projects/go-lib/src/lib/components/go-button/go-button.component.ts

```diff
@@ -9,21 +9,18 @@ export class GoButtonComponent {
   @Input() buttonIcon: string;
   @Input() buttonType: string = 'button';
   @Input() buttonVariant: string;
-  @Input() useLoader: boolean = false;
+  @Input() isProcessing: boolean = false;
   @Input() useDarkTheme: boolean = false;
+  @Input() useLoader: boolean = false;
```

Comment:
Leaving this here, so that it won't give compiler errors when introducing this change to an app that already uses it. In a later version we can completely deprecate it. This way it gives people time to get away from it instead of just breaking all of the templates that use it